Formula/mmseqs2.rb

```diff
@@ -0,0 +1,63 @@
+class Mmseqs2 < Formula
+  desc "Software suite for very fast protein sequence search and clustering"
+  homepage "https://mmseqs.org"
+  url "https://github.com/soedinglab/MMseqs2/archive/1-c7a89.tar.gz"
+  version "1-c7a89"
+  sha256 "e756a0e5cb3aa8e1e5a5b834a58ae955d9594be1806f0f32800427c55f3a45d5"
+
+  head do
+    url "https://github.com/soedinglab/MMseqs2.git"
+    resource "documentation" do
+      url "https://github.com/soedinglab/MMseqs2.wiki.git"
+    end
+  end
+
+  bottle :disable, "Requires SSE4.1 support"
+
+  option "with-avx2", "Build with AVX2 instruction set support"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on :arch => :x86_64
+  depends_on :mpi => [:cxx, :optional]
```

Comment:
decide if you want this in or out but the formula should have no options. also this has been deprecated in favor of just `depends_on "gcc"`

Reply:
Could you clarify please what I should change here?

You mean changing `depends_on "gcc" => :build` to `depends_on "gcc"`?

I would also like to keep optional the MPI dependency/option.

Reply:
Right `depends_on "gcc"`

>I would also like to keep optional the MPI dependency/option.

The formula should have no options so turn it on by default :)

Reply:
There are a few downsides to having MPI support enabled by default:
1) We do some additional (unnecessary for single node) merge steps of the result files
3) Some CLI flags for a low-overhead mode are only available in non-mpi mode
2) Non-mpi is better tested by us